vendor/wp-now/src/wp-now.ts

```diff
@@ -158,6 +129,55 @@ export default async function startWPNow(
 	};
 }
 
+async function getPHPInstance( options: WPNowOptions, isPrimary: boolean, requestHandler: PHPRequestHandler ) : Promise<{ php: PHP; runtimeId: number }> {
+	const id = await loadNodeRuntime( options.phpVersion );
+	const php = new PHP(id);
+
+	await setPhpIniEntries(php, {
+		'memory_limit': '256M',
+		'disable_functions': '',
+		'allow_url_fopen': '1',
+		'openssl.cafile': '/internal/shared/ca-bundle.crt'
+	});
+
+	return { php, runtimeId: id };
+}
+
+function prepareDocumentRoot( php: PHP, options: WPNowOptions ) {
+	php.mkdir(options.documentRoot);
+	php.chdir(options.documentRoot);
+	php.writeFile(
+		`${options.documentRoot}/index.php`,
+		`<?php echo 'Hello wp-now!';`
+	);
+	php.writeFile(
+		'/internal/shared/ca-bundle.crt', rootCertificates.join('\n')
+	)
+}
+
+async function prepareWordPress( php: PHP, options: WPNowOptions 	) {
```

Comment:
Nitpick: there's a tab at the end of the argument list.
```suggestion
async function prepareWordPress( php: PHP, options: WPNowOptions ) {
```